pages/common/brew-cask.md

```diff
@@ -1,6 +1,7 @@
 # brew cask
```

Comment:
It's a little confusing to have `brew cask` as command name, but then document `brew --cask`. I think we should rather completely replace `brew cask` with `brew --cask` and add a note like ```This command previously was called as `brew cask`, but has been deprecated in favor of `brew --cask` ```.

Reply:
Agreed, I will update it tomorrow. Bye